src/lib.rs

```diff
@@ -343,7 +345,8 @@ mod test;
 #[cfg(feature = "tracing-unstable")]
 mod trace;
 
-#[cfg(feature = "in-use-encryption-unstable")]
+#[cfg(any(feature = "in-use-encryption-unstable", docsrs))]
+#[cfg_attr(docsrs, doc(cfg(any(feature = "sync", feature = "tokio-sync"))))]
```

Comment:
I think the second line should be
```
#[cfg_attr(docsrs, doc(cfg(feature = "in-use-encryption-unstable")))]
```
(and below).

Reply:
whoops, bad copy/paste - fixed